frontend/src/types/plan.ts

```diff
@@ -92,7 +92,10 @@ export const FEATURE_MATRIX: Map<FeatureType, boolean[]> = new Map([
   // Database management
   ["bb.feature.pitr", [false, true, true]],
   ["bb.feature.read-replica-connection", [false, false, true]],
-  ["bb.feature.sync-schema", [true, true, true]],
+  // This feature type is specifically means that all schema versions can be selected.
+  // Sync schema is free to all plans. But in non-enterprise plan, we only show the
+  // first schema version and it's not selectable.
+  ["bb.feature.sync-schema", [false, false, true]],
```

Comment:
Do we need to rename this into `bb.feature.sync-schema-with-all-versions`? @tianzhou

Reply:
Good idea.

Reply:
bb.feature.sync-schema-all-versions